pom.xml

```diff
@@ -339,6 +339,7 @@
           </execution>
         </executions>
       </plugin>
+      <!-- clenaup artifact sign 
```

Comment:
What's this needed for?

Reply:
somehow my local compilation was failing on signing the artifact...I need to install some plugin...will fix it